src/enforester.js

```diff
@@ -63,7 +63,7 @@ const EXPR_LOOP_OPERATOR = {};
 const EXPR_LOOP_NO_CHANGE = {};
 const EXPR_LOOP_EXPANSION = {};
 
-function getLineNumber(x: Syntax | T.Term) {
+function getLineNumber(x: any) {
```

Comment:
because flow doesn't pass otherwise 😅 
 
My plan was to just get flow to pass and then refactor to better types in a separate PR but looking at the diff now it's probably better to do the refactoring now.

I'll add some more commits so we don't regress on the types.